hack/graph.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+#
+# Usage:
+#
+#   graph.sh [URI] >graph.dot
+#
+# For example:
+#
+#   graph.sh https://cincinnati.example.org/your/graph | dot -Tsvg >graph.svg
+set -e
+
+URI="${1:-https://api.openshift.com/api/upgrades_info/v1/graph?channel=prerelease-4.1}"
```

Comment:
I was thinking of a way to make this URI a template with a nested variable for the channel as this is very guaranteed to change in the future. However that thought was superseded by the comment below, which solves the issue for this script altogether.

Reply:
Obsolete after [this][1].

[1]: https://github.com/openshift/cincinnati/pull/108#discussion_r288158349